docs/ref/databases.txt

```diff
@@ -719,7 +719,7 @@ you add quotes where necessary before copying a query into an SQLite shell.
 Oracle notes
 ============
 
-Django supports `Oracle Database Server`_ versions 11.1 and higher. Version
+Django supports `Oracle Database Server`_ versions 11.2 and higher. Version
```

Comment:
Thanks for the explanation. Completed the docs for dropping support for 11.1 in https://github.com/django/django/pull/5310.